src/taxes/components/TaxChannelsPage/TaxChannelsPage.tsx

```diff
@@ -0,0 +1,99 @@
+import { Card } from "@material-ui/core";
+import VerticalSpacer from "@saleor/apps/components/VerticalSpacer";
+import CardTitle from "@saleor/components/CardTitle";
+import Container from "@saleor/components/Container";
+import Grid from "@saleor/components/Grid";
+import PageHeader from "@saleor/components/PageHeader";
+import { sectionNames } from "@saleor/intl";
+import {
+  Button,
+  List,
+  ListHeader,
+  ListItem,
+  ListItemCell,
+  PageTab,
+  PageTabs
+} from "@saleor/macaw-ui";
+import { taxesMessages } from "@saleor/taxes/messages";
+import React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+
+import TaxChannelsMenu from "../TaxChannelsMenu";
+import TaxCountryExceptionListItem from "../TaxCountryExceptionListItem";
+import TaxSettingsCard from "../TaxSettingsCard";
+
+interface TaxChannelsPageProps {
+  data: any;
+  channels: any;
```

Comment:
`any` in types is not very nice
please don't use any 😅 - types for channels should be generated from GraphQL, I don't know what `data` is for